src/test/java/school/lemon/changerequest/java/junit/hw1/task1/api/ContactApiTest.java

```diff
@@ -0,0 +1,215 @@
+package school.lemon.changerequest.java.junit.hw1.task1.api;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import school.lemon.changerequest.java.junit.hw1.task1.model.*;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomNumeric;
+import static org.junit.Assert.*;
+
+public class ContactApiTest {
+
+    ContactApi contactApi = null;
+    Contact testContact = null;
+
+    @Before
+    public void setUp() {
+        contactApi = new ContactApiImpl(new ContactIdGeneratorImpl());
+        testContact = new Contact((long) 0, new CallNumber("+1", "2345", "6789012"), "Alex", "Marukhnenko");
```

Comment:
Do you really need to create your test object before each test? Can it be `private static final` one?
What about having the other test variable — `testCallNumber`?